src/screens/UserPortal/UserScreen/UserScreen.spec.tsx

```
@@ -81,15 +83,47 @@ vi.mock('utils/useSession', () => ({
-  default: vi.fn(() => (
-    <div data-testid="profile-dropdown">
-      <div data-testid="display-name">Test User</div>
+vi.mock('shared-components/DropDownButton', () => ({
+  default: vi.fn((props) => (
+    <div
+      data-testid="user-profile-dropdown"
+      data-variant={props.variant}
+      data-menu-class={props.menuClassName}
+      data-show-caret={props.showCaret}
+      aria-label={props.ariaLabel}
+    >
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)